apps/theme/app/page.tsx

```diff
@@ -162,7 +162,7 @@ export default function Home() {
         colorModalRef={colorModalRef}
       />
 
-      <main className={classes.main}>
+      <main className={classes.main} id="main">
```

Comment:
```suggestion
      <main className={classes.main} id='main'>
```

Reply:
Thanks!

// note to self, `yarn biome ci .` to lint locally